packages/components/src/components/card/card.jsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const CardRoot = ({ children, className, ...props }) => (
+    <div className={classNames('dc-card__wrapper', className)} {...props}>
+        {children}
+    </div>
+);
+
+const CardHeader = ({ children }) => <div className='dc-card__header-wrapper'>{children}</div>;
+
+const CardContent = ({ children }) => <div className='dc-card__content-wrapper'>{children}</div>;
+
+const CardFooter = ({ children }) => <div className='dc-card__footer-wrapper'>{children}</div>;
+
+const Card = ({ className, header, content, footer, renderHeader, renderContent, renderFooter }) => {
```

Comment:
Is it possible to pipe the `{...props}` to `CardRoot`?

Reply:
Yes I will do for better usability of the component, thanks for the advice.